src/docs/zksync-era.mdx

```diff
@@ -119,8 +119,13 @@ links:
     | 43 | NUMBER | | Returns the latest L2 block's number | Returns the latest block’s number <Modified /> |
```

Comment:
There is a difference in the behavior of SELFDESTRUCT in the Ethereum L1 Behavior column